integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java

```diff
@@ -125,13 +125,7 @@ public static OpenLineageYaml extractOpenlineageConfFromSparkConf(SparkConf conf
   }
 
   private static List<Tuple2<String, String>> filterProperties(SparkConf conf) {
-    return Arrays.stream(conf.getAllWithPrefix("spark.openlineage."))
```

Comment:
The removal of properties was necessary because the `OpenLineageYaml` object mappers were improperly initialized without disabling `FAIL_ON_UNKNOWN_PROPERTIES`. I've fixed that, after that the removal here was unnecessary.